registry/config.go

```diff
@@ -67,7 +67,7 @@ var lookupIP = net.LookupIP
 // InstallCliFlags adds command-line options to the top-level flag parser for
 // the current process.
 func (options *ServiceOptions) InstallCliFlags(flags *pflag.FlagSet) {
-	mirrors := opts.NewNamedListOptsRef("registry-mirrors", &options.Mirrors, ValidateMirror)
```

Comment:
Since every mirror in the daemon no matter from flag, config or reload config, they will all be validated in `LoadMirrors`.
While ValidateMirror twice will lead to error, since the first `ValidateMirror` will add `/` at the end of the mirror address, if the new mirror enter the second `ValidateMirror` with a postfix of `/`, it will report error.